buildkite/bazel-central-registry/bcr_presubmit.py

```diff
@@ -170,6 +170,7 @@ def create_anonymous_repo(module_name, module_version, root=None):
     scratch_file(root, ".bazelrc", [
         "build --enable_bzlmod",
         "build --registry=%s" % BCR_REPO_DIR.as_uri(),
+        "query --registry=%s" % BCR_REPO_DIR.as_uri(),
```

Comment:
Thanks! Let's change the two "build" above to "common" instead, then those two flags should apply to all commands, like `cquery`, `aquery` etc.